my-app/src/location.js

```diff
@@ -1,23 +1,25 @@
 export function geoFindMe() {
```

Comment:
I am a bit confused about the intent of this change. The main concern I have is that I think latitude and longitude can both be negative and -1 and -2 are both valid values.

Reply:
I think one possible solution could be to return a third variable, status

Reply:
Oh yeah sorry the -1 and -2 are placeholders tbh. I just needed to assign a value there or else if errors. This was done so I could remove the long and lat from display.js without any errors